gapic-generator/templates/default/service/rest/client/_config.erb

```diff
@@ -87,6 +87,15 @@ class Configuration
   config_attr :metadata,      nil, ::Hash, nil
   config_attr :retry_policy,  nil, ::Hash, ::Proc, nil
   config_attr :quota_project, nil, ::String, nil
+  <%- if service.rest.mixin_should_generate_override_config? -%>
+
+  # @private
+  # Overrides for http bindings for the RPCs of this service
+  # are only used when this service is used as mixin, and only
+  # by the host service.
+  # @return [::Hash{::Symbol=>::Array{::Gapic::Rest::GrpcTranscoder::HttpBinding}}]
```

Comment:
Use `<>` instead of `{}` to surround the array element types. See https://yardoc.org/types.html

Reply:
:+1: